src/openrct2/ride/TrackPaint.cpp

```diff
@@ -29,15 +29,15 @@
 
 // clang-format off
 /* rct2: 0x007667AC */
-static LocationXY16 loc_7667AC[] = {
+static TileCoordsXY EntranceOffsetEdgeNE[] = {
```

Comment:
Can use constexpr here.

Reply:
Can't currently, since `TileCoordsXY(int32_t, int32_t)` is not `constexpr` (unlike `TileCoordsXYZ(int32_t, int32_t, int32_t)`. I suppose this is an error, I'll change that too

Reply:
There's a bunch of `const CoordsXY(Z)` declarations in this file - should I also change these to `constexpr`s?

Reply:
There's a bunch of `const CoordsXY(Z)`s in this file, should I change these to `constexpr`s too?